tests/stdlib/tlists.nim

```diff
@@ -81,3 +81,39 @@ block tlistsToString:
     l.append('2')
     l.append('3')
     doAssert $l == """['1', '2', '3']"""
+
+block SinglyLinkedListConversion:
```

Comment:
name the block after the symbols you're testing: `toSinglyLinkedList`

it makes it easier to search for tests for a given symbol
(note that merely using a symbol in some test isn't the same as a dedicated block testing it)